sdk/maintenance/Azure.ResourceManager.Maintenance/src/Custom/RestOperations/MaintenanceConfigurationsRestOperations.cs

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+using Azure.Core.Pipeline;
+using Azure.ResourceManager.Maintenance.Models;
+
+namespace Azure.ResourceManager.Maintenance
+{
+    internal partial class MaintenanceConfigurationsRestOperations
+    {
+        internal HttpMessage CreateDeleteRequest(string subscriptionId, string resourceGroupName, string resourceName)
```

Comment:
Any reason that we need to customize this method? I don't see the difference between this and the generated one.

Reply:
I added comments for why customize this operation:

// In the swagger this delete operation is defined to returen a `MaintenanceConfigurationData` model, but actually the service doesn't return, that cause exception when try to descerialize the response payload .
// As this lib has already GAed, so here using custom code to retrieve the data before running the delete operation that can keep backward compatibility.

Reply:
I agree that we should customize the method `public async Task<Response<MaintenanceConfigurationData>> DeleteAsync(...`, but this method, aka `internal HttpMessage CreateDeleteRequest(...`, is same as the generated code. So I don't think we should remove it from the generated directory to the custom directory.

Reply:
Fixed, use `directive` to fix the swagger spec and customize the `Delete` operations in the  `MaintenanceConfigurationResource` .